unime/src/lib/secure-logging.test.ts

```diff
@@ -0,0 +1,49 @@
+import { sanitizeStringify } from './secure-logging';
+
+describe('secure-logging', () => {
+  test('Check if secure logged', () => {
+    let credentialObj = {
+      name: 'John doe',
+      age: 44,
+      password: 'please_dont_log',
+    };
+
+    let sampleStr = JSON.stringify(credentialObj);
+
+    let secureStr = sanitizeStringify(credentialObj);
```

Comment:
I know it's "just tests", but I feel like the term _secure_ has a different meaning than _sensitive_. This PR doesn't add to the security of the application, but to keep the sensitive values out of the logs. So I think we should also use that wording accordingly.